jquery.dynatable.js

```diff
@@ -259,7 +259,11 @@
       // that may have been in URL when page loaded, so that it doesn't conflict with
       // what's passed in with the data ajax parameter
       } else {
-        options.url = utility.refreshQueryString(window.location.href, {}, this.settings);
+        var ajaxUrl = utility.refreshQueryString(window.location.href, {}, this.settings);
+        if (ajaxUrl.length > 0) {
+          ajaxUrl = '?' + ajaxUrl;
+        }
```

Comment:
Can you explain what these lines are trying to do? Should the path be prepended here?